provisioning/device/devdoc/x509_registration_requirements.md

```diff
@@ -19,7 +19,9 @@ The `register` method completes the authentication and registration flow for the
 
 **SRS_NODE_DPS_X509_REGISTRATION_18_001: [** `register` shall call `getCertificate` on the security object to acquire the X509 certificate. **]**
 
-**SRS_NODE_DPS_X509_REGISTRATION_18_002: [** `register` shall call `registerX509` on the transport object and call it's callback with the result of the transport operation. **]**
+***SRS_NODE_DPS_X509_REGISTRATION_18_004: [** `register` shall pass the certificate into the `setAuthentication` method on the transport **]**
+
+**SRS_NODE_DPS_X509_REGISTRATION_18_002: [** `register` shall call `register` on the pollingStateMachine and call `callback` with the result. **]**
```

Comment:
>SRS_NODE_DPS_X509_REGISTRATION_18_002 [](start = 2, length = 37)

I think we're missing a requirement about what happens when registration fails (calls the callback with an error)